driver/driver.go

```diff
@@ -228,6 +228,9 @@ type UI interface {
 	// WantBrowser indicates whether browser should be opened with the -http option.
 	WantBrowser() bool
 
+	// DisableBrowser forces WantBrowser() to return false.
+	DisableBrowser()
```

Comment:
I wouldn't want to make changes to this interface unless it's absolutely required. Can we instead make the value of the "no browser" flag available to webui.go and handle the decision of not opening the browser there?

Reply:
Sure. I figured you'd prefer to have "WantBrowser" return the truthful value, but it's just as easy to pass it through as a bool.

Reply:
Done.